cicd/forgeops-tests/Dockerfile

```diff
@@ -3,6 +3,10 @@ FROM python:3.5-alpine
 
 RUN apk add --no-cache bash
 
+RUN pip3 install pytest
+RUN pip3 install allure-pytest
+RUN pip3 install pytest-html
+RUN pip3 install pytest-metadata
+RUN pip3 install requests
 RUN pip install requests
-RUN pip install html-testRunner
 RUN pip install beautifulsoup4
```

Comment:
should this be ```RUN pip3 install beautifulsoup4```

Reply:
Good catch, should be `pip3` in this file